simapp/simd/cmd/root.go

```diff
@@ -56,11 +55,12 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		Short: "simulation app",
 		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
 
+			initClientCtx = initClientCtx.WithHomeFlag(cmd)
```

Comment:
Should I move WithHomeFlag(cmd) inside config.ReadFromClientConfig to make our code a little bit cleaner?